client/my-sites/plans-v2/plans-filter-bar/style.scss

```diff
@@ -75,7 +75,7 @@
 	.segmented-control {
 		margin-left: 8px;
 		margin-right: 16px;
-		flex: 0 0 170px;
+		flex: 0 0 200px;
```

Comment:
I adjusted this to `flex-basis: 0 0 200px`, because I think this is a better average starting width required by most languages.